### PR TITLE
feat(frontend): ask user for notification when they connect

### PR DESCRIPTION
**Pull request by knodelle**
Issue : #5455 

## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [x] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)
- [x] Make sure that:
  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified
- [x] Original authors of the features included in a multi-feature branch (maintenance fixes -> develop, security fixes -> develop, …) should be part of the reviewers, especially if you encountered merge conflicts.

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [ ] Manual, quality tests have been done

### CODE REVIEW

File: frontend/src/container/Login.jsx [hunk 1]
```diff
@@ -268,6 +268,20 @@ class Login extends React.Component {
 
   handleUserConnection = async () => {
     const { props } = this
+
+    if (window.Notification) {
+      try {
+        if (Notification.permission !== 'denied') {
+          const permission = Notification.requestPermission()
+          if (permission === 'granted') {
+            new Notification('notification granted') // eslint-disable-line no-new
```
Comment (gbonaspetti) [inline frontend/src/container/Login.jsx:277]: Should 'notification granted' be translated? I don't know if this `if` is not a forgotten debug

**Comment by gbonaspetti** [comment]
This PR should be from/to maintenance/4.1

File: frontend/src/container/Login.jsx (hunk 1 above)
Comment (knodelle) [reply frontend/src/container/Login.jsx:277]: Yes, we probably don't need to create a notification here, how about I just delete this condition?

**Review by gbonaspetti — APPROVED** [review]

**Comment by knodelle** [comment]
This branch has been rebased here #5467

**Pull request closed without merging**